ingest-app/src/cmr/ingest/api/subscriptions.clj

```diff
@@ -60,6 +62,30 @@
            subscriber-id
            concept-id)))))
 
+(defn- search-gran-refs-with-sub-params
+  "Performs a granule search using the provided query parameters. If the query is no good,
+   we throw a service error."
+  [context params]
+  (try
+    (search/find-granule-references context params)
```

Comment:
Since this function is called on every subscription ingest, we should not use the try catch to handle a normal operation scenario. Let's create a new function (maybe called `find-granule-references-return-errors`) to return the error from search regularly rather than throwing exception, and call this new function here.